src/dashboard/src/components/charts/allOrgDonut/useDonutChart.ts

```diff
@@ -0,0 +1,28 @@
+import { useFiltered } from '@/store';
+import React from 'react';
+import { IStats } from './IStats';
+import { defaultData } from './defaultData';
+import { updateData } from './updateData';
+
+export const useDonutChart = () => {
```

Comment:
If this is also useful for other chart component I will extract to a better location.